package.json

```diff
@@ -35,6 +35,7 @@
     "@html-next/vertical-collection": "3.0.0-0",
     "broccoli-string-replace": "^0.1.2",
     "css-element-queries": "^0.4.0",
+    "ember-angle-bracket-invocation-polyfill": "^3.0.1",
```

Comment:
Martin mentioned that we should always include the polyfill. Newer versions of Ember will not transform.